spec/models/expired_vacancy_feedback_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
```

Comment:
This could be deleted if the Expired Vacancy Feedback migration stuff is removed